src/test/reducerTests/reducer.test.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-unused-expressions */
+import { filterData } from '../../redux/countries/api';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.